industrial_robot_client/launch/robot_interface_download.launch

```diff
@@ -30,6 +30,9 @@
   <node pkg="industrial_robot_client" type="motion_download_interface" name="motion_download_interface"/>
   
   <!-- joint_trajectory_action: provides actionlib interface for high-level robot control -->
-  <node pkg="industrial_robot_client" type="joint_trajectory_action" name="joint_trajectory_action"/>
+  <node pkg="industrial_robot_client" type="joint_trajectory_action" name="joint_trajectory_action">
+    <param name="ignore_motion_server_error" type="bool" value="true" />
```

Comment:
```suggestion
    <param name="ignore_motion_server_error" type="bool" value="false" />
```
I really appreciate this being tunable but I think we should be strict by default as it can introduce surprising delayed movements if you later enable movements (maybe  ... maybe not ... ?)

Reply:
So this is indeed the main point of discussion here.

Doing this by default has the potential to break ppl's setups.

We could merge this PR as-is and not enable it on Melodic, then branch for Noetic and enable it there by default.

I hate branching for these little things, but I don't really see another option.

Reply:
If we leave it disabled, how would you enable it manually? copy paste the launch file?

I totally agree on avoiding branch-mess where possible

Reply:
I see now that I forgot to export the `param`s using `arg`s. I'll add that. That would allow overriding the defaults.

Reply:
Seems I've forgotten to fix the `arg`s missing. I'll need to address that before merging this.